core/src/main/java/com/hcl/ecomm/core/models/Listmodel.java

```diff
@@ -0,0 +1,33 @@
+package com.hcl.ecomm.core.models;
+
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+@Model(adaptables = Resource.class)
+public class Listmodel {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    @Inject
+    @Optional
+    private String list;
```

Comment:
@Rajamouli-hcl why String variable is named list?

Reply:
because in the dialog i have given in the multifield the textfiled name as list  so i am getting the input  values from the list variable